typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aiconfig",
-  "version": "1.1.13",
+  "version": "1.1.14",
```

Comment:
fwiw, I don't think we need to keep bumping this each week when there are no changes